packages/www/pages/dashboard/streams/[id]/health.tsx

```diff
@@ -0,0 +1,201 @@
+        d="M24 17L31 24L24 31"
+        stroke={active ? "#943CFF" : "#CCCCCC"}
+        strokeWidth="2"
+        strokeLinecap="round"
+        strokeLinejoin="round"
+      />
+    </svg>
+  );
+};
+
+type Props = {
+  active: boolean;
+};
+
+const interval = 10000;
+const maxItems = 6;
+
+const Health = () => {
+  const [stream, setStream] = useState(null);
+  const [dataChart, setDataChart] = useState<{ name: number; kbps: number }[]>([
+    { name: 0, kbps: 0 },
+  ]);
+  const [info, setInfo] = useState<StreamInfo | null>(null);
+  const [videoExists, setVideoExists] = useState<boolean>(false);
+
+  const { getStream, getStreamInfo } = useApi();
+  const router = useRouter();
+  const { query } = router;
+  const id = query.id;
+
+  const doGetInfo = useCallback(
+    async (id: string) => {
+      setInfo(null);
+      const [, rinfo] = await getStreamInfo(id);
+      if (!rinfo || rinfo.isSession === undefined) {
+        return;
+      } else if (rinfo.stream) {
+        const info = rinfo as StreamInfo;
+        setInfo(info);
+      }
+    },
+    [getStreamInfo]
+  );
+
+  const getIngestRate = useCallback(
+    async (id: string) => {
+      const [, rinfo] = await getStreamInfo(id);
+      if (!rinfo) {
+        return;
+      } else if (rinfo.stream) {
+        const newInfo = rinfo as StreamInfo;
+        setDataChart((prev) => {
+          const lastItem = prev[prev.length - 1];
+          return [
+            ...prev,
+            {
+              name: lastItem ? lastItem.name + interval / 1000 : 0,
+              kbps: Math.round(newInfo.session.ingestRate / 1000),
+            },
+          ].slice(Math.max(prev.length - maxItems, 0));
+        });
+      }
+    },
+    [getStreamInfo]
+  );
+
+  useEffect(() => {
+    if (!id) {
+      return;
+    }
+    getStream(id)
+      .then((stream) => setStream(stream))
+      .catch((err) => {
+        console.error(err);
+      }); // todo: surface this
+    doGetInfo(typeof id === "string" ? id : null);
+  }, [doGetInfo]);
+
+  useEffect(() => {
+    const intervalId = window.setInterval(() => {
+      if (info) {
+        getIngestRate(typeof id === "string" ? id : null);
+      } else return null;
+    }, interval);
+
+    return () => {
+      window.clearInterval(intervalId);
+    };
+  }, [getIngestRate, info]);
+
+  const playbackUrl = `https://cdn.livepeer.monster/hls/${stream?.playbackId}/index.m3u8`;
+
+  if (!stream) {
+    return (
+      <Layout
+        id="streams"
+        breadcrumbs={[
+          { title: "Streams", href: "/dashboard/streams" },
+        ]}></Layout>
+    );
+  }
+
+  return (
+    <Layout
+      id="streams"
+      breadcrumbs={[
+        { title: "Streams", href: "/dashboard/streams" },
+        { title: stream?.name, href: `/dashboard/streams/${id}` },
+        { title: "Stream Health" },
+      ]}>
+      <Box css={{ padding: "48px 34px" }}>
+        <Flex
+          css={{
+            borderBottom: "1px solid rgba(0, 0, 0, 0.15)",
```

Comment:
Border color should be `$mauve6`. Eg: `borderBottom: "1px solid $colors$mauve6"`. This will ensure correct color when using dark vs light mode.

Let's also try and use space tokens where we can (eg `px: $3`)